coretex/cli/commands/project.py

```diff
@@ -20,47 +20,106 @@
 import click
 
 from ..modules import ui, project_utils, utils, user
-from ...entities import Project
-from ...networking import NetworkRequestError
-from ...configuration import loadConfig, saveConfig
+from ..modules.ui import clickPrompt, errorEcho, successEcho, progressEcho
+from ...entities import Project, ProjectType, ProjectVisibility
+from ...networking import NetworkRequestError, EntityNotCreated, networkManager, RequestFailedError
+from ...configuration import loadConfig
 
 
-def selectProject(projectId: int) -> None:
+@click.command()
+@click.option("--project", "-p", type = str, help = "Project name")
```

Comment:
Call the option `--name` instead of `--project` when creating a project

Reply:
Didn't we want to use only -p or --project flag as part of coretex project flow task? (CTX-5531)

Reply:
That is for existing projects, this is creating a new project. Its `coretex project create --project some_name` vs `coretex project create --name some_name`, and the second one feels more intuitive